crates/pipeline-types/src/transport/delta_table.rs

```diff
@@ -3,11 +3,32 @@ use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 use utoipa::ToSchema;
 
+/// Delta table write mode.
+///
+/// Determines how the Delta table connector handles an existing table at the target location.
+#[derive(Default, Debug, Clone, Eq, PartialEq, Deserialize, Serialize, ToSchema)]
```

Comment:
Do we be need all these traits?

Reply:
yes, I think there's a good reason for each of them.